plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -2076,6 +2087,9 @@ private static void postNvpConfigBeforeStart(VirtualMachineMO vmMo, VirtualMachi
                 } else if (backing instanceof VirtualEthernetCardNetworkBackingInfo) {
                     // This NIC is connected to a Virtual Switch
                     // Nothing to do
+                } else if (backing instanceof VirtualEthernetCardOpaqueNetworkBackingInfo) {
+                    //if NSX API VERSION >= 4.2, connect to br-int (nsx.network), do not create portgroup else previous behaviour
+                    //OK, connected to OpaqueNetwork
```

Comment:
Those 2 else/if blocks should contain some logging in order to inform sys admins what is now comments in the code.

You can keep it for now, because it doesn't affect the feature that @miguelaferreira has already tested. Once this PR is merged, we can fix it.